src/components/backgroundselector/BackgroundSelectorDirective.js

```diff
@@ -78,6 +74,29 @@ goog.require('ga_topic_service');
             });
             return isValid;
           };
+          var updateDefaultBgOrder = function(bgLayers) {
+            bgLayers = bgLayers ? bgLayers : [];
+            defaultBgOrder = [];
+            bgLayers.forEach(function(bgLayerId) {
+              if (bgLayerId === voidLayer.id) {
+                defaultBgOrder.push(voidLayer);
+              } else {
+                defaultBgOrder.push({
+                  id: bgLayerId,
+                  label: gaLayers.getLayerProperty(bgLayerId, 'label')
```

Comment:
We have here an issue. We have different names form the layer name and the background layer name. bg layer labels are bg_luftbild, bg_pixel_color, bg_pixel_grey. How to handle this name here ?